pom.xml

```diff
@@ -610,6 +610,12 @@ Service-Component: OSGI-INF/*.xml
                 <version>5.10.0</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
Let us rather manage all versions together with the junit-bom (like outlined in https://github.com/junit-team/junit5-samples/blob/efac0e2c2f59eff4b147f67145010d890e062f55/junit5-jupiter-starter-maven/pom.xml#L18).

Reply:
Makes sense. This should be fixed now.